tests/phpunit/tests/blocks/getHookedBlocks.php

```diff
@@ -96,55 +96,40 @@ public function test_get_hooked_blocks_matches_found() {
 			)
 		);
 
-		$this->assertSame(
+		$this->assertEquals(
```

Comment:
Why is this being changed to `assertEquals()` from `assertSame()` ?

Reply:
Ah, I missed that in my review. I'll see about changing it back!

Reply:
675686a012